### PR TITLE
[ZEPPELIN-577] ASCII control character errors in Zeppelin

### PR DESCRIPTION
**Pull request by minahlee**
### What is this PR for?
Take care of `interpreter not found` issue when code is copied from CRLF line ending editor

### What type of PR is it?
Bug Fix

### What is the Jira issue?
[ZEPPELIN-577](https://issues.apache.org/jira/browse/ZEPPELIN-577)

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java [hunk 1]
```diff
@@ -143,7 +143,7 @@ public static String getRequiredReplName(String text) {
     int scriptHeadIndex = 0;
     for (int i = 0; i < text.length(); i++) {
       char ch = text.charAt(i);
-      if (ch == ' ' || ch == '\n' || ch == '(') {
```
Comment (felixcheung) [inline zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java]: should we change this to regex check for whitespace? I mean, what if someone put a tab there?

Comment (minahlee) [inline zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java]: thank you for review, addressed it in 3354a6e

**Comment by felixcheung** [comment]
LGTM

**Comment by Leemoonsoo** [comment]
LGTM

**Comment by prabhjyotsingh** [comment]
LGTM

**Comment by Leemoonsoo** [comment]
Merging into master

**Pull request closed without merging**